airflow/migrations/versions/b0125267960b_merge_004c1210f153_and_74effc47d867.py

```diff
@@ -24,9 +24,9 @@
 
 """
 
-from alembic import op
 import sqlalchemy as sa
 
+from alembic import op
```

Comment:
I thought I updated the template used to generate the migrations in #6216 but I obviously missed this case :(

Fancy updating the config like https://alembic.sqlalchemy.org/en/latest/autogenerate.html#basic-formatter-configuration using black or [zimports](https://pypi.org/project/zimports/) in another PR?